percentage.sh

```
@@ -0,0 +1,3 @@
+#! /bin/bash
+
+Percentage=a%b
```

Shellcheck (0.10.0)

[warning] Percentage appears unused. Verify use (or export if used externally).

(SC2034)